templates/frontend/pages/aboutEditorialTeamBio.tpl

```diff
@@ -21,10 +25,21 @@
 		{$pageTitle|strip_unsafe_html}
 	</h2>
 
-	<div class="editorialTeamBioContent">
-		{$editor->getLocalizedData('biography')|strip_unsafe_html}
+	<div>
+		{if $profileImage}
+			<img src="{$baseUrl}/{$publicfiles}/{$profileImageUpload}?{""|uniqid}" alt="{translate key="user.profile.form.profileImage"}" />
+		{/if}
+		{$editor->getLocalizedData('biography')|strip_unsafe_html}<br>
```

Comment:
The addition of a `<br />` here is suspect.  If there are spacing issues, the correct way to deal with this is adding CSS for margin, padding, etc.